### ISSUE TITLE
Leflair - Full-Stack Developer - Saigon - FT

### ISSUE THREAD

**Issue by angiedang** [awesome-jobs/vietnam#366]
## About Us

Flaunt Your Full-Stack Flair at the coolest ecommerce startup in town!!!

Leflair is the go-to destination for smart buyers who wish to purchase branded and quality products at great price, while enjoying the same exclusive shopping experience as in luxury stores.
Leflair provides its members with exclusive access to the world’s best brands at up to 50% off retail. Every morning, we begin flash sales for women, men, kids, and home.

## Location

* District 4, Saigon, Vietnam

## Salary Expectation

* up to $2000/month 

## Requirements
* Angular2 / Javascript
* Node.js
* HTML5, CSS3, Bootstrap
* Responsive Design
* Git
* Automated Testing

### Must Have

* Be able to fly (just kidding)
* Strong English skills
* Attention to detail
* Love startup and the craziness of it
* Total awesomeness

### Ideally

* MongoDB/Mongoose
* Grunt, Gulp etc.
* CI/CD Tools
* Cloud Infrastructure and Docker
* Microservices Architecture

### Responsibilities

* Work with the CTO (Australian expat) to build our ecommerce web site and management platform
* Implement site designs using HTML5, CSS3, Bootstrap, AngularJS
* Develop our backend APIs using Node.js
* Work with NoSQL and SQL databases
* Test our applications
* Code review and refactoring

### You’ll be responsible for

* Basically everything technical at Leflair

### You can bring

* Your sense of humor
* Your unicorn

### Contact

Apply on our Career Page at www.leflair.co/careers

![leflair vietnam founders](https://cloud.githubusercontent.com/assets/22335722/20215814/922ff636-a849-11e6-9f33-ca579716abde.png)

**Comment by joneslee85** [comment]
👏 great job post, MUCH better than previous one.

**Comment by angiedang** [comment]
Thanks @joneslee85 :+1: we keep learning ;)

**Comment by vuongtran** [comment]
Greate job! +1

**Comment by angiedang** [comment]
@vuongtran I sent you an email some time ago ;)

**Comment by nvs2394** [comment]
Great job +1

**Comment by angiedang** [comment]
still looking for some more awesomeness...

**Comment by angiedang** [comment]
check out who's in our Tech team ;)

https://www.facebook.com/pg/leflairtuyendung/photos/?tab=album&album_id=1376122402418752
